website/source/docs/agent/services.html.markdown

```diff
@@ -73,6 +74,25 @@ from `1`.
 
 Note: there is more information about [checks here](/docs/agent/checks.html). 
 
+The `enableTagOverride` can optionally specified to disable the antientropy feature for 
+this service. If `enableTagOverride` is set to TRUE then external agents can
+reregister this service and modify the tags. Subsequent local sync operations
```

Comment:
Do you maybe want to mention "catalog" here?

_If `enableTagOverride` is set to TRUE then external agents can update this service in the catalog and modify the tags._